cnd/src/bitcoin_fees.rs

```diff
@@ -11,3 +11,142 @@ impl BitcoinFees {
         Ok(self.0)
     }
 }
+
+mod blockcypher {
+    use ::bitcoin::Amount;
+    use anyhow::Result;
+    use serde::Deserialize;
+
+    #[derive(Debug, Clone)]
+    pub struct Client {
+        url: url::Url,
+        network: comit::Network,
+    }
+
+    /// https://www.blockcypher.com/dev/bitcoin/#blockchain
+    impl Client {
+        /// Full URL to access blockcypher API blockchain endpoint: http://api.blockcypher.com/v1/btc/main
+        pub fn new(url: url::Url, network: comit::Network) -> Self {
+            Self { url, network }
```

Comment:
Should we choose the URL based on the network?

Reply:
Oh, good idea.